src/Containers/ContributionForm/ContributionForm.js

```diff
@@ -77,6 +79,15 @@ function ContributionForm() {
     }
   }
 
+  const makeList = () => {
+    if(specialties) {
+      const splitList = specialties.split(', ')
+      return splitList
+    } else {
+      return []
+    }
+  }
+
```

Comment:
This seems like a nice way to handle having multiple items. Later on we may want to refactor the form to help prevent sad paths